src/testing/Cargo.toml

```diff
@@ -4,7 +4,22 @@ version = "0.1.0"
 edition = "2021"
 publish = false
 
+[[example]]
+name = "googletest-example"
+crate-type = ["staticlib"]
```

Comment:
Why a staticlib? Are you planning to link it into C code?

Reply:
This was the type used as an example in [Examples](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#examples). It allows me to avoid an empty (and unwanted) `main` function.